ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,78 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        machine_config_selector (dict): Matching labels/expressions, to determine which MachineConfig objects
+            to apply this MachineConfigPool object.
+            For filtering based on labels, the `matchLabels` dict is used - the same way as it is used in the
+                nodeSelector (see the example of node_selector["matchLabels"] below).
+            For filtering based on expressions, the `matchExpressions` dict is used - the same way as it is used in the
+                nodeSelector (see the example of node_selector["matchExpressions"] below).
```

Comment:
```suggestion
        machine_config_selector (dict): Matching labels/expressions. <add example here>
```

Reply:
Instead of duplicating the same example, I added an explicit reference to the examples that I added for the nodeSelector members.